packages/components/src/h-stack/stories/index.story.tsx

```diff
@@ -42,7 +42,8 @@ const JUSTIFICATIONS = {
 
 const meta: Meta< typeof HStack > = {
 	component: HStack,
-	title: 'Components (Experimental)/HStack',
+	title: 'Components (Experimental)/Layout/HStack',
```

Comment:
Experimental components will fall in their own section down below, meaning they will have their own Layout directory. Is that desired? Should we consider marking the experimental components just with badges, like `CustomSelectControlV2` is marked WIP?

cc @ciampo @mirka